spec/factories/category.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :category, class: Category do
+    name        "main course"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.